src/extension.ts

```diff
@@ -490,7 +490,7 @@ export function deactivate() {
     getTelemetryLogger().dispose();
   } catch (e) {
     const msg = "Error disposing telemetry logger during extension deactivation";
-    logError(new Error(msg, { cause: e }), msg, {}, true);
+    logError(new Error(msg, { cause: e }), msg, { extra: {} });
```

Comment:
Nothing sensitive that needs to be handled here, but would also be a good use case for a "send to sentry" value ⬆️